centaur/src/main/resources/standardTestCases/three_step/three_step.cwl

```diff
@@ -37,7 +31,7 @@ steps:
       # Check it: this DockerRequirement does not use the `class` formatting but should still parse
       # thanks to the magic of SALAD.
       DockerRequirement:
-        dockerPull: "debian:wheezy"
+        dockerPull: "ubuntu:bionic"
```

Comment:
The `ps` command specifies `ubuntu:bionic` as one of its `requirements` and gets that. `cgrep` and `wc` don't express `hints` or `requirements`, but there is a `hints` at the workflow level that requests `ubuntu:latest`.